hoot-services/src/main/java/hoot/services/utils/DbUtils.java

```diff
@@ -216,6 +218,14 @@ public static long updateMapsTableTags(Map<String, String> tags, long mapId) {
                 .execute();
     }
 
+    public static void updateFoldersMapping() {
+        createQuery().insert(folderMapMappings).columns(folderMapMappings.mapId, folderMapMappings.folderId)
+                .select(new SQLQuery<>().select(maps.id, Expressions.numberTemplate(Long.class, "0")).from(maps)
+                        .where(maps.id.notIn(new SQLQuery<>().select(folderMapMappings.mapId).distinct()
+                                .from(folderMapMappings))))
+                .execute();
+    }
+
```

Comment:
Consolidate with queries in FolderResource, maybe in a FolderManger class
https://github.com/ngageoint/hootenanny/blob/3070/hoot-services/src/main/java/hoot/services/controllers/osm/map/FolderResource.java#L354-L358